def cause_type_bug():
    a = 10
    b = "5"
    print(a + b)  # TypeError: int + str
